src/Goerli-DssSpell.t.base.sol

```diff
@@ -0,0 +1,2327 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity 0.6.12;
+
+import "ds-math/math.sol";
+
+import "ds-test/test.sol";
+import "dss-interfaces/Interfaces.sol";
+
+import "./test/rates.sol";
+import "./test/addresses_goerli.sol";
+
+import {DssSpell} from "./Goerli-DssSpell.sol";
+
+interface Hevm {
+    function warp(uint256) external;
+    function store(address,bytes32,bytes32) external;
+    function load(address,bytes32) external view returns (bytes32);
+}
+
+interface SpellLike {
+    function done() external view returns (bool);
+    function eta() external view returns (uint256);
+    function cast() external;
+    function nextCastTime() external returns (uint256);
+}
+
+interface AuthLike {
+    function wards(address) external view returns (uint256);
+}
+
+interface PsmAbstract {
+    function wards(address) external returns (uint256);
+    function vat() external returns (address);
+    function gemJoin() external returns (address);
+    function dai() external returns (address);
+    function daiJoin() external returns (address);
+    function ilk() external returns (bytes32);
+    function vow() external returns (address);
+    function tin() external returns (uint256);
+    function tout() external returns (uint256);
+    function file(bytes32 what, uint256 data) external;
+    function sellGem(address usr, uint256 gemAmt) external;
+    function buyGem(address usr, uint256 gemAmt) external;
+}
+
+interface BrokeTokenAbstract {
+    function name() external view returns (bytes32);
+    function symbol() external view returns (bytes32);
+    function decimals() external view returns (uint256);
+    function totalSupply() external view returns (uint256);
+    function balanceOf(address) external view returns (uint256);
+    function transfer(address, uint256) external;
+    function allowance(address, address) external view returns (uint256);
+    function approve(address, uint256) external;
+    function approve(address) external;
+    function transferFrom(address, address, uint256) external;
+    function push(address, uint256) external;
+    function pull(address, uint256) external;
+    function move(address, address, uint256) external;
+    function mint(uint256) external;
+    function mint(address,uint) external;
+    function burn(uint256) external;
+    function burn(address,uint) external;
+    function setName(bytes32) external;
+    function authority() external view returns (address);
+    function owner() external view returns (address);
+    function setOwner(address) external;
+    function setAuthority(address) external;
+}
```

Comment:
We could move the interfaces in `dss-interfaces` repo to reduce space (`SpellLike`,` AuthLike` and `PsmAbstract`).
`BrokenTokenAbstract` I think could be removed as it was specific to #19 (USDT-A Offboarding).

Reply:
i'll create another PR to handle that so this doesn't get too big